internal/app/singularity/instance_linux.go

```diff
@@ -23,28 +24,59 @@ type instanceInfo struct {
 	Pid      int    `json:"pid"`
 	Image    string `json:"img"`
 	IP       string `json:"ip"`
+	LogErr   string `json:"logErr"`
+	LogOut   string `json:"logOut"`
 }
 
 // PrintInstanceList fetches instance list, applying name and
 // user filters, and prints it in a regular or a JSON format (if
-// formatJSON is true) to the passed writer.
-func PrintInstanceList(w io.Writer, name, user string, formatJSON bool) error {
+// formatJSON is true) to the passed writer. Additionally, fetches
+// log paths (if showLogs is true).
+func PrintInstanceList(w io.Writer, name, user string, formatJSON bool, showLogs bool) error {
+	if formatJSON && showLogs {
+		sylog.Fatalf("more than one flags have been set")
+	}
+
+	tabWriter := tabwriter.NewWriter(os.Stdout, 0, 8, 2, ' ', 0)
```

Comment:
I'd encourage you to use a `defer tabWriter.Flush()` immediately following `tabwriter.NewWriter()` (and remove the two calls below). This ensures the output is always flushed, regardless of which `return` is hit, and [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)s the code.